contrib/fast-import/import-zips.py

```diff
@@ -19,7 +19,7 @@
     exit(1)
 
 if len(argv) < 2:
-    print 'usage:', argv[0], '<zipfile>...'
+    print('usage:', argv[0], '<zipfile>...')
```

Comment:
Looks like this file needs `from __future__ import print_function` in case it is run from python 2

Reply:
Done.